packages/transaction-manager/test/unit/utils/test-data.ts

```diff
@@ -140,11 +140,15 @@ export class FakeEpkCipherProvider implements CipherProviderTypes.ICipherProvide
   public async encrypt(
     data: string,
     options: { encryptionParams: EncryptionTypes.IEncryptionParameters },
-  ): Promise<string> {
+  ): Promise<EncryptionTypes.IEncryptedData> {
     const encryptionParams = options.encryptionParams;
 
     if (encryptionParams.method === EncryptionTypes.METHOD.ECIES) {
-      return ecEncrypt(encryptionParams.key, data);
+      const encryptedValue = await ecEncrypt(encryptionParams.key, data);
+      return {
+        type: EncryptionTypes.METHOD.ECIES,
+        value: encryptedValue,
+      };
```

Comment:
This change synchronizes the mock code with the actual encrypt function. :+1: 
https://github.com/RequestNetwork/requestNetwork/blob/959aa97a4965fb8e4b6663f20f07960cbd4afb78/packages/utils/src/encryption.ts#L46-L56

Reply:
Sorry, is there anything I should change here?

Reply:
No change is necessary! Sorry, I should have made that clear in my original comment. This part confused me so I wanted to comment as a form of documentation.